assert/assert_is_error.ts

```diff
@@ -23,11 +23,11 @@ export function assertIsError<E extends Error = Error>(
   error: unknown,
   // deno-lint-ignore no-explicit-any
   ErrorClass?: new (...args: any[]) => E,
-  msgIncludes?: string,
+  msgIncludes?: string | RegExp,
   msg?: string,
 ): asserts error is E {
   const msgSuffix = msg ? `: ${msg}` : ".";
-  if (error instanceof Error === false) {
+  if (!(error instanceof Error)) {
```

Comment:
This allows type narrowing so we don't have to do more checks later in the function (see line 42 in the old diff)